vMenu/LanguageManager.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MenuAPI;
+using Newtonsoft.Json;
+using CitizenFX.Core;
+using static CitizenFX.Core.UI.Screen;
+using static CitizenFX.Core.Native.API;
+using static vMenuClient.CommonFunctions;
+using static vMenuShared.ConfigManager;
+using static vMenuShared.PermissionsManager;
+
+namespace vMenuClient
+{
+    public class LanguageManager : BaseScript
```

Comment:
Why does this use BaseScript?
Please make it a static class instead with static functions.